### PR TITLE
Docker builds, entrypoints improvements

### PR DESCRIPTION
**Pull request by tomaszdurka**
- Add example packages binaries

### CODE REVIEW

File: examples/api/package.json [hunk 1]
```diff
@@ -2,26 +2,23 @@
   "name": "@wepublish/api-example",
   "version": "0.0.0",
   "private": true,
-  "bin": {
-    "wepublish-api-example": "./dist/index.js"
-  },
   "main": "./dist/",
   "license": "MIT",
+  "bin": {
```
Comment (Itrulia) [inline examples/api/package.json:7]: do we maybe want to do `"bin": "./bin/run-server.js"`?

File: examples/oauth2/package.json [hunk 2]
```diff
@@ -2,18 +2,18 @@
   "name": "@wepublish/oauth2-example",
   "version": "0.0.0",
   "private": true,
-  "bin": {
-    "wepublish-oauth2-example": "./dist/index.js"
-  },
   "main": "./dist/",
   "license": "MIT",
+  "bin": {
```
Comment (Itrulia) [inline examples/oauth2/package.json:7]: Same as above

File: examples/website/package.json [hunk 3]
```diff
@@ -8,6 +8,9 @@
     "node": "^12.0.0",
     "yarn": "^1.16.0"
   },
+  "bin": {
+    "wepublish-example-website": "./bin/run-server.js"
```
Comment (Itrulia) [inline examples/website/package.json:12]: idem

File: packages/api/prisma/seed-base.ts [hunk 4]
```diff
@@ -0,0 +1,140 @@
+import {PrismaClient} from '@prisma/client'
```
Comment (Itrulia) [inline packages/api/prisma/seed-base.ts:1]: Documentation has to be updated now. This also makes a breaking change.

**Review by Itrulia — CHANGES_REQUESTED** [review]

File: packages/api/prisma/seed-base.ts (hunk 4 above)
Comment (tomaszdurka) [reply packages/api/prisma/seed-base.ts:1]: That was something I wanted to discuss. Did that in the last moment as could not make seed run without example.

File: packages/api/prisma/seed-base.ts [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+      id: 'peer',
+      systemRole: true,
+      name: 'Peer',
+      description: 'Peer Role',
+      permissionIDs: []
+    }
+  })
+]
+
+const seedUsers = async (prisma: PrismaClient) => [
```
Comment (Itrulia) [inline packages/api/prisma/seed-base.ts:116]: Should be moved out into a different seeding file, see prisma documentation

**Review by elias-summermatter — APPROVED** [review]

**Review by elias-summermatter — CHANGES_REQUESTED** [review]
Tests also not working

File: Dockerfile [hunk 6]
```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/wepublish/node:latest
+FROM ghcr.io/wepublish/node:latest as production
```
[the remark this answers is not in the document]
Comment (Itrulia) [reply Dockerfile:1]: Might our custom node images be the reason it takes around 3 minutes for the images to start once pulled by gke?

[the remark this answers is not in the document]
Comment (tomaszdurka) [reply Dockerfile:1]: It is using official just adding stuff on top of it.
https://github.com/wepublish/node-docker-base/blob/main/dockerfile

**Pull request merged**